A43-sdk-authorization.md

```diff
@@ -0,0 +1,713 @@
+A43: SDK authorization
+----
+* Author(s): [Ashitha Santhosh](https://github.com/ashithasantosh)
+* Approver: [Doug Fawley](https://github.com/dfawley),
+            [Eric Anderson](https://github.com/ejona86),
+            [Mark Roth](https://github.com/markdroth),
+            [Sanjay Pujare](https://github.com/sanjaypujare),
+            [Yihua Zhang](https://github.com/yihuazhang)
+* Status: Draft {Draft, In Review, Ready for Implementation, Implemented}
+* Implemented in: <language, ...>
+* Last updated: 2021-08-30
+* Discussion at: https://groups.google.com/g/grpc-io/c/m4Lns0tO-fE
+
+## Abstract
+
+gRPC is currently implementing an open-source standard authorization solution.
+This proposal discusses how this framework would be made available to OSS gRPC
+users (like network switches) to perform per-RPC authorization checks.
+
+## Background
+
+Authentication verifies the identity of the requestor. Authorization ensures that
+the requestor has sufficient access to make a particular request.
+
+gRPC offers built-in authentication mechanisms like TLS, ALTS and even allow
+users to plug in their own authentication systems. However, gRPC doesn't have any
+standard authorization solution available. If a user wanted to perform per RPC
+authorization check, they have to implement their own solution in the application.
+The standard gRPC authorization mechanism described in this proposal allows users
+to simply provide an authorization policy for their server. Then, for each RPC,
+the server verifies whether the client is authorized to issue the RPC.
```

Comment:
Changed the text to "server verifies whether the client is authorized to make the RPC request."

Since the statement starts off as `server verifies`, I think this works. Let me know if you think we should still change it. And I can work on the wording:)